README.md

```diff
@@ -122,6 +137,28 @@ To run kaniko in Docker, run the following command:
 ```shell
 ./run_in_docker.sh <path to Dockerfile> <path to build context> <destination of final image>
 ```
+### Pushing to Different Registries
+
+kaniko uses Docker credential helpers to push images to a registry.
+
+kaniko comes with support for GCR, but configuring another credential helper should allow pushing to a different registry.
```

Comment:
Could there be an example here of "configuring another credential helper"... or a link to documentation elsewhere that explains this ... I don't quite understand how this would work...

Reply:
Sure, I could look into it, could you open an issue for it? Thanks!